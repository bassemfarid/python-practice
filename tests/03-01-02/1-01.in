One
Two
Three
Four
Five
Six
Seven
Eight
Nine
A girl has no name