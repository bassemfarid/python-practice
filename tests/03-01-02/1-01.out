hello One!
NEXT!
hello Two!
NEXT!
hello Three!
NEXT!
hello Four!
NEXT!
hello Five!
NEXT!
hello Six!
NEXT!
hello Seven!
NEXT!
hello Eight!
NEXT!
hello Nine!
NEXT!
*sweats profusely*